opentrons-ai-client/src/pages/Settings/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { useAtom } from 'jotai'
+import { css } from 'styled-components'
+
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  Btn,
+  COLORS,
+  DIRECTION_COLUMN,
+  DIRECTION_ROW,
+  Flex,
+  Icon,
+  JUSTIFY_CENTER,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+
+import { FeatureFlag, Privacy } from '../../components/organisms/Settings'
+import { featureFlagsAtom } from '../../resources/atoms'
+
+const SETTINGS_MAX_WIDTH = '56rem'
+
+export function Settings(): JSX.Element {
+  const { t } = useTranslation('protocol_generator')
+  const navigate = useNavigate()
+  const [featureFlags, setFeatureFlags] = useAtom(featureFlagsAtom)
+
+  const prereleaseModeEnabled = featureFlags.enablePrereleaseMode === true
+
+  const handleToggleAnalytics = (): void => {
+    const currentValue = featureFlags.enableAnalytics ?? true
+    const newValue = !currentValue
+    setFeatureFlags({
+      enableAnalytics: newValue,
+    })
+  }
+
+  const handleTogglePDProtocolGeneration = (): void => {
+    const currentValue = featureFlags.enablePDProtocolGeneration ?? true
+    const newValue = !currentValue
+    setFeatureFlags({
+      enablePDProtocolGeneration: newValue,
+    })
+  }
+
+  const handleBackClick = (): void => {
+    navigate('/')
```

Comment:
This works fine when the user accesses it from the landing page, but it causes issues when they navigate to the Settings page from any other page.


https://github.com/user-attachments/assets/c6da478a-8277-4ad0-b45e-94c560dbda48



```suggestion
    navigate(-1)
```

Reply:
Nice Thanks!